sdk/ml/azure-ai-ml/azure/ai/ml/_schema/automl/automl_vertical.py

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-from azure.ai.ml._restclient.v2022_06_01_preview.models import LogVerbosity
+from azure.ai.ml._restclient.v2022_10_01_preview.models import LogVerbosity
```

Comment:
I notice you're making changes to the _schema folder in this repo. Is _schema not just for CLI? I thought CLI changes were supposed to be made in the pre-migration repo.